ui/src/dashboards/components/DashboardPage.tsx

```diff
@@ -46,8 +47,16 @@ const dashRoute = `/${ORGS}/${ORG_ID}/${DASHBOARDS}/${DASHBOARD_ID}`
 
 @ErrorHandling
 class DashboardPage extends Component<Props> {
+  public componentDidmount() {
+    if (isFlagEnabled('queryCacheForDashboards')) {
+      resetQueryCache()
+    }
+  }
+
   public componentWillUnmount() {
-    this.props.resetCachedQueryResults()
```

Comment:
doesn't this need to called if the flag is disabled?

Reply:
Not since the file has been removed and we're no longer caching things. Even if the feature flag is implemented the previous caching system has been ripped out in this PR

Reply:
that's what i'm trying to call attention to. doesn't that mean that we have interface changes being deployed that aren't captured / contained by the feature flag?